app/src/main/java/com/antonioleiva/bandhookkotlin/data/mapper/ArtistMapper.kt

```diff
@@ -17,28 +17,21 @@
 package com.antonioleiva.bandhookkotlin.data.mapper
 
 import com.antonioleiva.bandhookkotlin.data.lastfm.model.LastFmArtist
-import com.antonioleiva.bandhookkotlin.data.lastfm.model.LastFmImage
-import com.antonioleiva.bandhookkotlin.data.lastfm.model.LastFmImageType
 import com.antonioleiva.bandhookkotlin.domain.entity.Artist
 
-class ArtistMapper {
+class ArtistMapper(val imageMapper: ImageMapper = ImageMapper()) {
 
     fun transform(artists: List<LastFmArtist>): List<Artist> {
-        return artists.filter { artistHasQualityInfo(it) }.map { transform(it) }
+        return artists.take(10).filter { artistHasQualityInfo(it) }.map { transform(it) }
```

Comment:
Artist without mbid that makes app unusable is in the end of the list.
This is a temporary solution of that problem, that is addressed more fundamentally here https://github.com/antoniolg/Bandhook-Kotlin/pull/16